.github/workflows/main.yml

```diff
@@ -8,60 +8,42 @@ on:
 
 jobs:
   test:
-    runs-on: ubuntu-18.04
```

Comment:
[note] `ubuntu-18.04` is out of the support.
See https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#choosing-github-hosted-runners